plugins/event_engine_plugin/event_engine_plugin.cpp

```diff
@@ -191,35 +203,40 @@ void event_engine_plugin_impl::accepted_block( const chain::block_state_ptr& sta
     ilog("Accepted block: ${block_num}", ("block_num", state->block_num));
 
     AcceptedBlockMessage msg(BlockMessage::AcceptBlock, state);
-    send_message(msg);
+    if (last_block != state->id) {
+        msg_id = 0;
+        last_block = state->id;
+    } else {
+        msg_id++;
+    }
+    send_message(msg, msg_id);
 }
 
 void event_engine_plugin_impl::irreversible_block(const chain::block_state_ptr& state) {
     ilog("Irreversible block: ${block_num}", ("block_num", state->block_num));
 
     BlockMessage msg(BlockMessage::CommitBlock, state);
-    send_message(msg);
+    send_message(msg, UINT32_MAX);
```

Comment:
What reason to set msg-id for irreversible to UINT32_MAX? Why not 0?

Reply:
@afalaleev Because it is after all trxs and `AcceptBlock` for corresponding block. Still keeping it same way.